apis/comms_api/routers/router.py

```diff
@@ -1,10 +1,8 @@
 from fastapi import APIRouter, status
 from fastapi.responses import Response
 
-
 router = APIRouter(prefix='/api/v1')
```

Comment:
Decided to use a single space everywhere, makes it much easier to decide what to do with them

Reply:
Although it may make the code easier to maintain and extend, we must ensure PEP8 compliance from the first developments.